apps/zadmin/management/commands/gen_loadtest_users.py

```diff
@@ -0,0 +1,57 @@
+import os
+from datetime import datetime
+from optparse import make_option
+
+from django.core.management.base import BaseCommand, CommandError
+
+from apps.users.models import UserProfile
+
+
+class Command(BaseCommand):
+    help = 'Generate some fake users that can be used for load testing.'
+
+    option_list = BaseCommand.option_list + (
+        make_option(
+            '--total',
+            action='store',
+            type='int',
+            default=10,
+            help='Total number of users to generate. Default: %default.'),
+        make_option(
+            '--file-dest',
+            action='store',
+            default=os.getcwd(),
+            help='Directory to write user/pass file to. Default: %default.'),
+    )
+
+    def handle(self, *args, **options):
+        fn = os.path.join(options['file_dest'], 'loadtest-users.txt')
+        if os.path.exists(fn):
+            raise CommandError(
+                'User file exists: {}. Delete or move it first.'.format(fn))
+        group_id = os.urandom(3).encode('hex')
+
+        with open(fn, 'w') as user_file:
+            print 'About to generate {} users; prefix={}'.format(
+                options['total'], group_id)
+            for index in range(options['total']):
+                user_id = '{}-{}'.format(group_id, index)
+                username = 'loadtest-{}'.format(user_id)
+                email = '{}@nowhere.org'.format(username)
```

Comment:
`nowhere.org` looks like a valid domain. Should we use something like `addons.mozilla.org`?